src/gei/Services/GhesVersionCheckerService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Threading.Tasks;
+using OctoshiftCLI.Extensions;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.Services
+{
+    public class GhesVersionCheckerService
```

Comment:
nit: I wouldn't suffix `Service`:
```suggestion
    public class GhesVersionChecker
```

Reply:
I changed this as you suggested, but I'm thinking maybe I should change it back. Everything else in our code has suffix's. Commands, Handlers, Factories, etc.